docs/tutorials/06_qubit_mappers.ipynb

```diff
@@ -0,0 +1,476 @@
+      "+ 0.12091263261776633 * ZIZI\n"
+     ]
+    }
+   ],
+   "source": [
+    "qubit_p_op = mapper.map(fermionic_op)\n",
+    "print(qubit_p_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19dd459f",
+   "metadata": {},
+   "source": [
+    "This has one major benefit for the case of problems in which we want to preserve the number of particles of each spin species; it allows us to remove 2 qubits, because the information in them becomes redundant.\n",
+    "Since Qiskit Nature arranges the qubits in block-order, such that the first half encodes the alpha-spin, and the second half the beta-spin information, this means we can remove the N/2-th and N-th qubit.\n",
+    "\n",
+    "To do this, you need to use the `QubitConverter` like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "7592d0fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "converter = QubitConverter(ParityMapper(), two_qubit_reduction=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bdce1e0",
+   "metadata": {},
+   "source": [
+    "In order for the reduction to work, you now also need to supply the number of particles:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "569a3b78",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.0523732457728605 * II\n",
+      "+ 0.39793742484317896 * IZ\n",
+      "- 0.39793742484317896 * ZI\n",
+      "- 0.01128010425623538 * ZZ\n",
+      "+ 0.18093119978423122 * XX\n"
+     ]
+    }
+   ],
+   "source": [
+    "qubit_op = converter.convert(fermionic_op, num_particles=problem.num_particles)\n",
+    "print(qubit_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b4b1bec",
+   "metadata": {},
+   "source": [
+    "### More advanced qubit reductions\n",
+    "\n",
+    "The `QubitConverter` also enables you to perform more advanced qubit reductions, which are based on finding Z2 symmetries in the Hilbert space of the qubit. A requirement for this to be useful, is that you know in which symmetry-subspace you need to look for your actual solution of interest. This can be a bit tricky, but luckily the problem classes of Qiskit Nature provide you with a utility to automatically determine that correct subspace.\n",
+    "\n",
+    "Here is how you can use this to your advantage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "4270e7cf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.0410931415166251 * I\n",
+      "- 0.7958748496863578 * Z\n",
+      "+ 0.1809311997842312 * X\n"
+     ]
+    }
+   ],
+   "source": [
+    "converter = QubitConverter(JordanWignerMapper(), z2symmetry_reduction=\"auto\")\n",
+    "\n",
+    "qubit_op = converter.convert(fermionic_op, sector_locator=problem.symmetry_sector_locator)\n",
```

Comment:
We only have this auto symmetry sector locator for Fermionic systems - ElectronicStructureProblem  - but I guess this is all this tutorial covers for now but maybe its worth stating anyway.